cs/Serializer/Visitors/VowpalWabbitStringVisitor.cs

```diff
@@ -8,12 +8,15 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics.Contracts;
 using System.Globalization;
 using System.Linq;
 using System.Text;
 using VW.Interfaces;
 using VW.Serializer.Interfaces;
 
+#pragma warning disable 1591
```

Comment:
http://stackoverflow.com/questions/25846205/compiler-warning-cs1591-how-to-show-that-warning-only-for-undocumented-methods

From: Luong Hoang [mailto:notifications@github.com]
Sent: Monday, September 14, 2015 2:53 PM
To: JohnLangford/vowpal_wabbit <vowpal_wabbit@noreply.github.com>
Cc: Markus Cozowicz <marcozo@microsoft.com>
Subject: Re: [vowpal_wabbit] Documentation (#754)


In cs/Serializer/Visitors/VowpalWabbitStringVisitor.cs<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fJohnLangford%2fvowpal_wabbit%2fpull%2f754%23discussion_r39431845&data=01%7c01%7cmarcozo%40064d.mgd.microsoft.com%7cbb9cb91b64794362b3c208d2bd35aa11%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=Lgw4xxjGTs4Z5DZMHgl2ML9nCub3h4%2bgzkfXlNk7sow%3d>:

>  using System.Globalization;

>  using System.Linq;

>  using System.Text;

>  using VW.Interfaces;

>  using VW.Serializer.Interfaces;

>

> +#pragma warning disable 1591

which warning is this?

—
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fJohnLangford%2fvowpal_wabbit%2fpull%2f754%2ffiles%23r39431845&data=01%7c01%7cmarcozo%40064d.mgd.microsoft.com%7cbb9cb91b64794362b3c208d2bd35aa11%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=w2zI8TpJnzhl%2brvIqDF9grkVyIIBUgluAVrIhMQZPxs%3d>.